terraform-state/main.tf

```diff
@@ -0,0 +1,69 @@
+terraform {
```

Comment:
I think this terraform-state folder could do with a readme to explain that this creates the s3 bucket that stores the state files...